play-json/jvm/src/main/scala/play/api/libs/json/JsonConfig.scala

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (C) 2009-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package play.api.libs.json
```

Comment:
Renamed from `JsonParserSettings`.

I ended up using sealed traits, which I called `JsonConfig`, `BigDecimalParseConfig`, and `BigDecimalSerializerConfig`.